examples/mp-metrics-gauge/README_pt.adoc

```diff
@@ -0,0 +1,153 @@
+= Métricas com MicroProfile `@Gauge` (Medida)
+
+:index-group: MicroProfile
+:jbake-type: page
+:jbake-status: published
+
+Esta é um exemplo sobre como utilizar as métricas de MicroProfile em TomEE.
+
+== Executando a aplicação:
+
+[source,bash]
+----
+$ mvn clean install tomee:run
+----
+
+Dentro da aplicação, tem um endpoint que vai fornecer a temperatura em celsios para o día.
```

Comment:
`celsius` instead of `celsios`.
`dia` instead of `día`.

Reply:
done